db/view/view.cc

```diff
@@ -46,6 +46,7 @@
 #include "db/system_distributed_keyspace.hh"
```

Comment:
> The virtual reader by constructing a multishard reader which reads the rows from shard zero, and then filtering it to get only the rows owned by the current shard.

This sentence is incomplete (at the start).

Reply:
fixed